src/main/java/me/ryanhamshire/GriefPrevention/EntityEventHandler.java

```diff
@@ -775,71 +790,68 @@ private void handleEntityDamageEvent(EntityDamageEvent event, boolean sendErrorM
         Firework firework = null;
         Entity damageSource = subEvent.getDamager();
 
-        if (damageSource != null)
```

Comment:
Removing this unnecessary null check (and one in the next method) makes the diff look horrible, sorry. May want to just side-by-side compare for this class.